cypress/e2e/inspectlet.spec.cy.ts

```diff
@@ -0,0 +1,20 @@
+describe(
+  'To test inspectlet integration',
+  () => {
+    beforeEach(() => {
+      cy.visit('/');
+    });
+
+    it('injection of code according to wid', ()=>{
```

Comment:
Test case here is not valid. You need to define 2 different test cases where:
- 'Should enable integration if key was provided' - Providing a key should pass `cy.get('.inspectlet_diagnostics').should('be.visible')` assertion.
- 'Should not enable integration if key was not provided' - Not providing a key should pass `cy.get('.inspectlet_diagnostics').should('not.exist')` assertion.